From 5759385fb2b1fafcb0073add3cf4dc7a918ac7f2 Mon Sep 17 00:00:00 2001
From: Shiji Yang <yangshiji66@outlook.com>
Date: Thu, 8 May 2025 20:19:15 +0800
Subject: [PATCH 05/16] MIPS: lantiq: xway: add prototype for
 ltq_get_cp1_base()

ltq_get_cp1_base() is an exported function, we must define
its prototype on header file. Fix warning:

arch/mips/lantiq/xway/vmmc.c:22:15: error: no previous prototype for 'ltq_get_cp1_base' [-Werror=missing-prototypes]
   22 | unsigned int *ltq_get_cp1_base(void)
      |               ^~~~~~~~~~~~~~~~

Signed-off-by: Shiji Yang <yangshiji66@outlook.com>
---
 arch/mips/include/asm/mach-lantiq/xway/lantiq_soc.h | 3 +++
 1 file changed, 3 insertions(+)

--- a/arch/mips/include/asm/mach-lantiq/xway/lantiq_soc.h
+++ b/arch/mips/include/asm/mach-lantiq/xway/lantiq_soc.h
@@ -102,5 +102,8 @@ extern void ltq_pmu_disable(unsigned int
 /* allow the ethernet driver to load a flash mapped mac addr */
 const u8* ltq_get_eth_mac(void);
 
+/* VMMC */
+extern unsigned int *ltq_get_cp1_base(void);
+
 #endif /* CONFIG_SOC_TYPE_XWAY */
 #endif /* _LTQ_XWAY_H__ */
